lib/styles/constants.dart

```diff
@@ -13,7 +13,7 @@ class AppColors {
   //Colors
   static const Color red = Color(0xFFFF3B00);
   static const Color green = Color(0xFF00CC00);
-  static const Color orange = Color(0xFFFB513B);
+  static const Color orange = Color(0xFFF3474D);
```

Comment:
Are we deviating from the orange color code provided in v2.5?

Reply:
Yep, we've decided to change the color in the designs